tests/custom-package-edit-test.js

```diff
@@ -125,11 +125,11 @@ describeApplication('CustomPackageEdit', () => {
 
     describe('entering invalid data', () => {
       beforeEach(() => {
-        return PackageEditPage.interaction
-          .once(() => PackageEditPage.dateRangeRowList().length > 0)
+        return PackageEditPage
+          .when(() => PackageEditPage.dateRangeRowList().length > 0)
           .do(() => {
```

Comment:
FWIW, these are the `.when().do()` blocks we should be able to get rid of. Below, `.dateRangeRowList(0)` is lazy, so it will only continue if the `0` indexed item exists.

Reply:
I got rid of as many `.when().do()`s as I could 😁